[2023-01-16T08:46:29.764+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-14T00:00:00+00:00 [queued]>
[2023-01-16T08:46:29.782+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-14T00:00:00+00:00 [queued]>
[2023-01-16T08:46:29.783+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-01-16T08:46:29.783+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 6
[2023-01-16T08:46:29.784+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-01-16T08:46:29.942+0000] {taskinstance.py:1304} INFO - Executing <Task(BashOperator): second_task> on 2021-08-14 00:00:00+00:00
[2023-01-16T08:46:29.945+0000] {standard_task_runner.py:55} INFO - Started process 2041 to run task
[2023-01-16T08:46:29.949+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-08-14T00:00:00+00:00', '--job-id', '51', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpesve45gt']
[2023-01-16T08:46:29.951+0000] {standard_task_runner.py:83} INFO - Job 51: Subtask second_task
[2023-01-16T08:46:30.092+0000] {task_command.py:389} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-08-14T00:00:00+00:00 [running]> on host c7f1bfaaf748
[2023-01-16T08:46:30.425+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-14T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-14T00:00:00+00:00
[2023-01-16T08:46:30.426+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-01-16T08:46:30.427+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2023-01-16T08:46:30.436+0000] {subprocess.py:86} INFO - Output:
[2023-01-16T08:46:30.438+0000] {subprocess.py:93} INFO - hey, I am task2 and will be running after task1!
[2023-01-16T08:46:30.438+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-01-16T08:46:30.538+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20210814T000000, start_date=20230116T084629, end_date=20230116T084630
[2023-01-16T08:46:30.685+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-01-16T08:46:30.701+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
